### PR TITLE
Redeeming hotfix

### LINKED ISSUE #1016: Redemption button not showing up
Hi there, 

It looks like we are having DAO specific redemption issues. 

I recently passed proposals in DAOfund, and none of them have a "Redeem" button: http://prntscr.com/opa24a

Same thing with PravduhDAO: http://prntscr.com/opa1wq

This is pretty urgent as we are trying to get partners loaded up (at least for DAOfund)... thanks

### LINKED ISSUE #1027: Redemption Error Scheme Registrar CuraDAO
Hi,

I'm trying to redeem a Scheme Registrar proposal from the CuraDAO but am getting an error.

I was able to redeem one of the Contribution Reward proposals form the CuraDAO yesterday, so I'm not sure what happened here. Thanks for the help! 

Screenshot of error + console: 
image

### PR DESCRIPTION
**Pull request by jellegerbrandy**
- show redeem button on all redeemable proposals, fixes #1016
- make it so that non-Cr proposals are redeemable,  should fix #1027 #1018

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,15 +57,15 @@
     "start": "webpack-dev-server --config webpack.dev.config.js",
     "start-dev-profiling": "cross-env NODE_ENV=dev-profiling API_URL=https://daostack-alchemy.herokuapp.com S3_BUCKET=daostack-alchemy-staging webpack-dev-server --config webpack.dev.config.js",
     "start-docker": "webpack-dev-server --config webpack.docker.config.js",
-    "start-prod": "cross-env NODE_ENV=production webpack-dev-server --config webpack.prod.config.js",
+    "start-prod": "cross-env NODE_ENV=production webpack-dev-server --config webpack.dev.config.js",
```
Comment (dkent600) [inline package.json]: What is the intention here?

File: src/components/Proposal/RedemptionsTip.tsx [hunk 2]
```diff
@@ -83,9 +83,13 @@ export default (props: IProps) => {
             </li> : ""
           }
           {contributionRewards["rep"] ? <li><Reputation reputation={contributionRewards["rep"]} totalReputation={dao.reputationTotalSupply} daoName={dao.name} /></li> : ""}
-          { /*
-            TOOD: add native token
-          */ }
+
```
Comment (dkent600) [inline src/components/Proposal/RedemptionsTip.tsx]: Is this change really needed for this PR?

File: package.json (hunk 1 above)
Comment (jellegerbrandy) [reply package.json]: The intention here is: start with "production settings" (i.e. connect to mainnet and prod subgraph) but use the  `dev` webpack (so we still have source maps and can use the devtools debugger)

it sort  of slipped into this PR, bc I needed to check if the fix works in the current production settings, but I think this is the most useful setting here.

**Comment by jellegerbrandy** [comment]
yes

File: package.json (hunk 1 above)
Comment (dkent600) [reply package.json]: I imagine this will significantly increase the bundle size.  Is that acceptable?

Comment (jellegerbrandy) [reply package.json]: this command is only meant for devs; it is not actually used in production

**Comment by jellegerbrandy** [comment]
closing this, we decided to merge this not as a hotfix, but together with the dev-to-master deployment
